### DISCUSSION TITLE
Looking grapejs developer

### DISCUSSION THREAD

**Discussion by abdulaleem659** [GrapesJS/grapesjs#6267] [General]
Hi,

We are developing a SaaS-based Learning Management System (LMS). We are looking to integrate Grapejs as the CMS functionality for this project.

I would be glad to connect with any agencies or individuals,  who can assist us with this initiative.

Thank you!

**Comment by RakulAgn** [comment]
Hi @abdulaleem659  I think I could help with this if you like connect with me on [linkedIn](https://www.linkedin.com/in/rakul-agn/) Dude
